src/execution/vm/bytecode_generator.cpp

```diff
@@ -3045,6 +3045,50 @@ void BytecodeGenerator::VisitBuiltinCteScanCall(ast::CallExpr *call, ast::Builti
       Emitter()->Emit(Bytecode::CteScanFree, iterator);
       break;
     }
+    case ast::Builtin::IterCteScanInit: {
+      // Execution context
+      LocalVar exec_ctx = VisitExpressionForRValue(call->Arguments()[1]);
+      auto *arr_type = call->Arguments()[2]->GetType()->As<ast::ArrayType>();
+      LocalVar col_oids = VisitExpressionForLValue(call->Arguments()[2]);
+      // Emit the initialization codes
+      Emitter()->CteScanIteratorInit(Bytecode::IterCteScanInit, iterator, exec_ctx, col_oids,
+                                     static_cast<uint32_t>(arr_type->Length()));
+      break;
+    }
+    case ast::Builtin::IterCteScanGetResult: {
+      LocalVar result = ExecutionResult()->GetOrCreateDestination(call->GetType());
+      Emitter()->Emit(Bytecode::IterCteScanGetResult, result, iterator);
+      break;
+    }
+    case ast::Builtin::IterCteScanGetReadTable: {
+      LocalVar read = ExecutionResult()->GetOrCreateDestination(call->GetType());
```

Comment:
call this variable something else, github is parsing this as the read system call